src/citrine/_utils/template_util.py

```diff
@@ -0,0 +1,61 @@
+from gemd.util.impl import recursive_flatmap
+from gemd.entity.object import (
+    ProcessSpec,
+    ProcessRun,
+    MaterialSpec,
+    MeasurementSpec,
+    MeasurementRun,
+)
+from gemd.entity.attribute import PropertyAndConditions
+
+
+def make_attribute_table(gems: list):
```

Comment:
```suggestion
def make_attribute_table(gems: List[DataConcepts]) -> List[Mapping[str, BaseValue]]:
```
Let's make the signature a bit more explicit.  These types will need to be imported too.